spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -325,14 +321,7 @@ const ConfigSteps = () => {
 					{
 						isLoggedIn ? <>
 							{
-								loaderForLogin ? <>
```

Comment:
Why is this loader being removed? Without this loader, there will be a sudden jumpy kinda effect in case of slow networks.

Reply:
Yeah, happy to chat about this.
So right now there's a problem when I clicked the auth button. If I close the popup window directly, this auth button stays in "loading" state, I can't retry my login.

So we can fix either:
1. Use an `setInterval` to monitor close of sub window and remove the "loading" state if close.
2. Like in this PR, just remove the "loading" state. I think there won't be "jumpy" for that screen is because in order to have "jumpy", you have to have `isLoggedIn` and `loaderForLogin` change in some states. I still don't see when will that skeleton shown, as if `isLoggedIn` is true, `loaderForLogin` won't be true right?

Reply:
We can remove the loading state for the content inside login step, but I still think we need the loader for the Authorize button. If not with loader, then something else to at least disable the button, otherwise the user can just spam click the Authorize button sending multiple requests.

Reply:
Yeah thanks @krazziekay , let's chat this in another PR. 
Revert some of the changes.